src/ui/shared/Button.tsx

```diff
@@ -0,0 +1,20 @@
+interface ButtonProps {
+    label: string; // The text displayed on the button
+    onClick?: () => void; // Optional: Click handler
+    className?: string; // Optional: Additional Tailwind or custom classes
+    ariaPressed?: boolean; // Optional: For accessibility
```

Comment:
```suggestion
    label: string;
    onClick?: () => void;
    className?: string;
    ariaPressed?: boolean;
```
redundant comments - these parameters are well named and self-explanatory :)